lib/features/auth/repositories/auth_repository.dart

```diff
@@ -493,11 +494,15 @@ class AuthRepositoyImpl with AuthRepository {
   Future<void> setUserProperties(UserExt newUserExt) {
     FirebaseCrashlytics.instance.setUserIdentifier(newUserExt.guid);
 
+    if (newUserExt.country == Country.us.countryCode) {
```

Comment:
you can simplify this by using
if(newUserExt.isUsUser)
 :)

Reply:
Good one, thanks!